Cargo.toml

```diff
@@ -75,13 +77,16 @@ neard = { path = "./neard"}
 
 testlib = { path = "./test-utils/testlib" }
 
+memory-tracker = { path = "./memory-tracker" }
+
 [profile.release]
-lto = true        # Enable full link-time optimization.
-codegen-units = 1 # Use only 1 codegen-unit to enable full optimizations.
+lto = false        # Enable full link-time optimization.
+codegen-units = 256 # Use only 1 codegen-unit to enable full optimizations.
```

Comment:
why do we change these two configs here?

Reply:
I disabled those settings to decrease build time from 5m to 17s on my local machine. Mozilla was dealing with similar issue for Firefox, they ended up disabling lto for dev machines, but they enabled it only for release builds:
https://bugzilla.mozilla.org/show_bug.cgi?id=1386371
here is their PR - https://hg.mozilla.org/integration/autoland/rev/d75f07915a48
I think we should do something similar.